From a4acf53336a482f50335d33b4f650a70c9243b7b Mon Sep 17 00:00:00 2001
From: Nezih Yigitbasi <nyigitbasi@netflix.com>
Date: Sat, 6 Feb 2016 11:41:21 -0800
Subject: [PATCH] PARQUET-509: Fix args passed to string format calls

This PR fixes the args passed to the `String.format()` call.

Author: Nezih Yigitbasi <nyigitbasi@netflix.com>

Closes #320 from nezihyigitbasi/debug_args and squashes the following commits:

43a6088 [Nezih Yigitbasi] Fix args passed to string format calls
---
 .../java/org/apache/parquet/hadoop/DirectCodecFactory.java    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/DirectCodecFactory.java b/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/DirectCodecFactory.java
index bb711daa29..d90ab51093 100644
--- a/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/DirectCodecFactory.java
+++ b/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/DirectCodecFactory.java
@@ -389,7 +389,7 @@ public Object makeObject() throws Exception {
             decompressorPool.returnObject(decom);
           } else {
             if (Log.DEBUG) {
-              LOG.debug(String.format(BYTE_BUF_IMPL_NOT_FOUND_MSG, "decompressor" + codec.getClass().getName()));
+              LOG.debug(String.format(BYTE_BUF_IMPL_NOT_FOUND_MSG, "decompressor", codec.getClass().getName()));
             }
           }
 
@@ -409,7 +409,7 @@ public Object makeObject() throws Exception {
             } else {
               supportDirectDecompressor = false;
               if (Log.DEBUG) {
-                LOG.debug(String.format(BYTE_BUF_IMPL_NOT_FOUND_MSG, "compressor" + codec.getClass().getName()));
+                LOG.debug(String.format(BYTE_BUF_IMPL_NOT_FOUND_MSG, "compressor", codec.getClass().getName()));
               }
             }
 
